torch/_dynamo/backends/debugging.py

```diff
@@ -173,12 +208,12 @@ def get_nop_func():
 # Useful for debugging purpose
 # aot_eager uses AOT Autograd backend with nop compiler. It is helpful in debugging.
 def aot_eager(
-    gm,
-    fake_tensor_inputs,
-    fw_compiler=None,
-    bw_compiler=None,
-    **kwargs,
-):
+    gm: torch.fx.GraphModule,
+    fake_tensor_inputs: list[torch.Tensor],
+    fw_compiler: Optional[Callable[..., Any]] = None,
```

Comment:
Does this have any relation to the output Callable that can be expressed?

Reply:
I believe these are just used internally; perhaps we can refine these more further with a type alias since it seems to be a common pattern